resources/styles/global/typography.less

```diff
@@ -1,92 +1,27 @@
-@googleFontApi: "//fonts.googleapis.com/css?family=";
-@fontPath: "/fonts/";
-
-@font-face {
-  font-family: 'Architects Daughter';
-  src: local('Architects Daughter'),
-       local('ArchitectsDaughter'),
-       // url("@{googleFontApi}Architects+Daughter"),
-       url("@{fontPath}ArchitectsDaughter.ttf") format('truetype');
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Light'),
-         // url("@{googleFontApi}Lato:300"),
-         url("@{fontPath}Lato-Light.ttf") format('truetype');
-    font-weight: 300;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-LightItalic'),
-         // url("@{googleFontApi}Lato:300italic"),
-         url("@{fontPath}Lato-LightItalic.ttf") format('truetype');
-    font-weight: 300;
-    font-style: italic, oblique;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Regular'),
-         // url("@{googleFontApi}Lato:400"),
-         url("@{fontPath}Lato-Regular.ttf") format('truetype');
-    font-weight: 400;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-RegularItalic'),
-         // url("@{googleFontApi}Lato:400italic"),
-         url("@{fontPath}Lato-RegularItalic.ttf") format('truetype');
-    font-weight: 400;
-    font-style: italic, oblique;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Bold'),
-         // url("@{googleFontApi}Lato:700"),
-         url("@{fontPath}Lato-Bold.ttf") format('truetype');
-    font-weight: 700;
-}
-
-@font-face {
-    font-family: "Lato";
-    src: local('Lato-Black'),
-         // url("@{googleFontApi}Lato:900"),
-         url("@{fontPath}Lato-Black.ttf") format('truetype');
-    font-weight: 900;
-}
-
-@font-face {
-  font-family: 'Merriweather';
-  src: local('Merriweather Light'),
-       local('Merriweather-Light'),
-       // url("@{googleFontApi}Merriweather:300"),
-       url("@{fontPath}Merriweather-Light.ttf") format('truetype');
-  font-weight: 300;
+.tutor-font-face(@name, @filename, @weight, @style:normal) {
+  @font-face {
+    font-family: @name;
+    src: local(@name),
+    url('fonts/@{filename}.woff2') format('woff2'),
+    url('fonts/@{filename}.woff') format('woff');
+    font-weight: @weight;
+    font-style: @style;
+  }
 }
 
```

Comment:
1) Oh, I see why they were there now.  @dak was using the same family name and setting the weight and style so the appropriate font would be selected when the weight and height matched.  I'll rework to use that method so they'll make sense.
http://stackoverflow.com/questions/28279989/multiple-font-weights-one-font-face-query

2) Sorry, I think you've reached the bounds of my less knowledge.  Do you mean the url() generation? If so I suppose we could, but then the caller would have to specify the complete filename for both files (woff and woff2).

I'm getting the font list from @darrenmason for inclusion.